.travis.yml

```diff
@@ -1,7 +1,19 @@
+env:
+  global:
+    - NODE_ENV=test
+    - secret = authorshaven
+    - DATABASE_TEST=postgres://postgres:admin5423/users_test
```

Comment:
I am having a security issue here, you **should not** pass our secret and URL in this script